ethToElrond/v2/ethToElrond/steps/step05WaitForQuorum.go

```diff
@@ -0,0 +1,39 @@
+package steps
+
+import (
+	"context"
+
+	"github.com/ElrondNetwork/elrond-eth-bridge/core"
+	"github.com/ElrondNetwork/elrond-eth-bridge/ethToElrond/v2/ethToElrond"
+)
+
+type waitForQuorumStep struct {
+	bridge ethToElrond.EthToElrondBridge
+}
+
+// Execute will execute this step returning the next step to be executed
+func (step *waitForQuorumStep) Execute(ctx context.Context) (core.StepIdentifier, error) {
+	isQuorumReached, err := step.bridge.IsQuorumReached(ctx)
+	if err != nil {
+		step.bridge.GetLogger().Error("error while checking the quorum", "error", err)
+		return ethToElrond.GetPendingBatchFromEthereum, nil
+	}
+
+	step.bridge.GetLogger().Debug("quorum reached check", "is reached", isQuorumReached)
```

Comment:
a batch ID log would be helpful

Reply:
the batchID is logged before, no need to log it again at this step.